testing/unit/tp/node_account/test_close_masternode.py

```diff
@@ -0,0 +1,180 @@
+"""
+Provide tests for node account handler close masternode implementation.
+"""
+import time
+
+import pytest
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from sawtooth_sdk.protobuf.processor_pb2 import TpProcessRequest
+from sawtooth_sdk.protobuf.transaction_pb2 import (
+    Transaction,
+    TransactionHeader,
+)
+
+from remme.protos.node_account_pb2 import (
+    NodeAccountMethod,
+    NodeAccount,
+    CloseMasternodePayload,
+)
+from remme.protos.transaction_pb2 import TransactionPayload
+from remme.shared.utils import hash512
+from remme.tp.node_account import NodeAccountHandler
+from testing.conftest import create_signer
+
+from .shared import (
+    RANDOM_NODE_PUBLIC_KEY,
+    NODE_ACCOUNT_ADDRESS_FROM,
+    NODE_ACCOUNT_FROM_PRIVATE_KEY,
+    INPUTS,
+    OUTPUTS,
+    TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS,
+    create_context,
+)
+
+FROZEN = 300000
+UNFROZEN = 10000
+OPERATIONAL = 5000
```

Comment:
Clarify constant. Ask yourself `frozen what?`. Frozen balance, frozen amount to transfer to, etc.

Reply:
Done. FROZEN_BALANCE.